chi_build/square.lua

```diff
@@ -0,0 +1,77 @@
+--############################################### Setup mesh
```

Comment:
This will probably become your first regression test. Move it to the `ChiTest` folder and rename it something like `CFEMDiffusion_Test1_2Dsquare.lua`.

Another thing is that this file was in `chi_build` and is a problem because if anybody runs `./configure.sh clean` then this file will be deleted.